src/e2e-test/resources/pluginParameters.properties

```diff
@@ -1,15 +1,23 @@
 projectId=cdf-athena
 dataset=test_automation
+bqTable=db_to_bq
```

Comment:
Why the bqTable is hardcoded here? We can get the table from the hook.

Reply:
done
commit id- https://github.com/cloudsufi/google-cloud/commit/defc54df110255ef539dea8d206481cc145b2fec